binding.gyp

```diff
@@ -24,20 +24,17 @@
                 ],
             }],
             [ "OS == 'linux' or OS == 'mac'", {
-                "libraries": [
-                    "-lpthread",
-                ],
-                "cflags": [
-                    "-std=gnu99",
-                ],
-
 		"libraries": [
-		    "/opt/libdeltachat/lib/libdeltachat.so",
+		    "<!(echo $LIBDELTACHAT_DIR/lib/libdeltachat.so)",
```

Comment:
What's wrong with using `pkg-config`? If you install libdeltachat to a different location, you can just adapt `PKG_CONFIG_PATH`.

Reply:
The question is, if LIBDELTACHAT_DIR is not more generic? Not sure how widespread pkg-config is.

Reply:
I would say pkg-config is more generic cause it can also set other options. And distros will probably prefer pkg-config because it means less manual work: Their package managers set up `PKG_CONFIG_PATH` automatically.

Reply:
Anyway, if you install lideltachat with CMake, you'll get `deltachat.pc` for free.